client/src/partials/debtors/groups.list.html

```diff
@@ -0,0 +1,70 @@
+<div class="container text-center">
+<div class="row">
+<div class="col-md-8 col-md-offset-2">
+
+    <div class="panel panel-default" style="margin-bottom : 0px;">
```

Comment:
Noting for future reference -- this should probably be a transcluded directive/component eventually.  Fine for an initial implementation though.